test/Global.test.hpp

```diff
@@ -1,19 +1,19 @@
-/// Apache License 2.0
-
-#include <gtest/gtest.h>
-
-namespace ostk
-{
-namespace astro
-{
-namespace test
-{
-namespace global
-{
-
-extern std::string someGlobalString;
-
-}
-}  // namespace test
-}  // namespace astro
-}  // namespace ostk
+/// Apache License 2.0
+
+#include <gtest/gtest.h>
+
+namespace ostk
+{
+namespace astro
+{
+namespace test
+{
+namespace global
+{
+
+extern std::string someGlobalString;
+
+}
+}  // namespace test
+}  // namespace astro
+}  // namespace ostk
```

Comment:
Not sure what happened to those 2 files when applying the script 🤔

Reply:
Yeah me neither.